app/models/va_facility.rb

```diff
@@ -8,8 +8,8 @@ class VaFacility < ApplicationRecord
 
   attr_accessor :reset_cached_va_facilities
 
-  scope :get_by_visn, -> (visn) { cached_va_facilities.where(visn: visn) }
-  scope :get_classification_counts, -> (facility_type) { where(classification: facility_type).size }
+  scope :get_by_visn, -> (visn) { cached_va_facilities.where(visn: visn, hidden: false) }
```

Comment:
we do not need to add the hidden false since it is already being filtered on cached va facilities

Reply:
Removed it...